### PR TITLE
Add data structure files from week 2 courses.

### PR DESCRIPTION
**Pull request by rzkibar**
week 2 python course 
• dictionary
• list
• tuple

### CODE REVIEW

**Review by ariefrahmansyah — CHANGES_REQUESTED** [review]
can you rename the file to `course_1.py`

using `#` as filename is not good practice

**Comment by rzkibar** [comment]
> can you rename the file to `course_1.py`
> 
> using `#` as filename is not good practice



> can you rename the file to `course_1.py`
> 
> using `#` as filename is not good practice

my bad.. udah diubah kak

**Comment by rzkibar** [comment]
> can you rename the file to `course_1.py`
> 
> using `#` as filename is not good practice

my bad.. udah diubah kak

File: rzkibar/list_compre [hunk 1]
```diff
@@ -0,0 +1,4 @@
+numbers = [10, 20, 30]
```
Comment (ariefrahmansyah) [inline rzkibar/list_compre]: you need to add .py extension here :)

File: rzkibar/list_looping [hunk 2]
```diff
@@ -0,0 +1,3 @@
+students = ["baba", "bibi", "bubu", "bobo"]
```
Comment (ariefrahmansyah) [inline rzkibar/list_looping]: you need to add .py extension here :)

**Review by ariefrahmansyah — CHANGES_REQUESTED** [review]

**Review by ariefrahmansyah — APPROVED** [review]
LGTM! Mantapp

**Pull request merged**